static/app/utils.tsx

```diff
@@ -339,3 +339,15 @@ export function escapeDoubleQuotes(str: string) {
 export function generateBaseControlSiloUrl() {
   return ConfigStore.get('links').sentryUrl || '';
 }
+
+export function generateIntegrationInstallLink(orgSlug) {
```

Comment:
Actually this function doesn't seem specific to integrations right? Looks like a general URL translator.

Reply:
You're right. This function just switches the domain to the other org, but keeps the URL pathname the same. Renamed to `generateOrgSlugUrl`, sounds good?